plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/point/common/common.ts

```diff
@@ -0,0 +1,9 @@
+export function array_move(arr: any[], old_index: number, new_index: number) {
```

Comment:
Can you search for this component. Nina added it to a utils folder somewhere. If it makes more snese to you to be here rather than where she put it, please remove hers and update imports that were using it.
If her placement is better, please remove this file.

Reply:
Done!